templates/map.html

```diff
@@ -662,6 +691,74 @@ <h4>{{ i18n('Export Pokémon lists') }}</h4>
                     </div>
                   </div>
                 </div>
+                {% if settings.highlightPokemon %}
+                <div class="form-control">
+                  <div class="switch">
+                    <label>
+                      <input type="checkbox" id="pokemon-highlight-switch">
+                      <span class="lever"></span>
+                      {{ i18n('Highlight Pokémon') }}
+                    </label>
+                  </div>
+                </div>
+                <div id="highlight-pokemon-wrapper">
+                  {% if settings.highlightPokemon != "server" %}
+                  <div class="form-control">
+                    <div class="input-field color">
+                      <div class="text-input-title">{{ i18n('Color for Perfect IV') }}</div>
+                      <input placeholder="{{ i18n('Color...') }}" id="highlight-color-perfect" type="color">
+                    </div>
+                  </div>
+                  <div class="form-control">
+                    <div class="input-field color">
+                      <div class="text-input-title">{{ i18n('Color for High IV') }}</div>
+                      <input placeholder="{{ i18n('Color...') }}" id="highlight-color-iv" type="color">
+                    </div>
+                  </div>
+                  <div class="form-control">
+                    <div class="input-field color">
+                      <div class="text-input-title">{{ i18n('Color for High Level') }}</div>
+                      <input placeholder="{{ i18n('Color...') }}" id="highlight-color-level" type="color">
+                    </div>
+                  </div>
+                  {% if settings.highlightPokemon == "svg" %}
+                  <div class="form-control">
+                    <span id="highlight-size-slider-title" class="slider-title">Circle Size</span>
+                    <div class="slider-container">
+                      <div id="highlight-size-slider"></div>
+                    </div>
+                  </div>
+                  {% endif %}
+                  <div class="form-control">
+                    <span id="highlight-radius-slider-title" class="slider-title">Blur Radius</span>
+                    <div class="slider-container">
+                      <div id="highlight-radius-slider"></div>
+                    </div>
+                  </div>
+                  {% endif %}
+                  <div class="form-control">
+                    <span id="highlight-iv-slider-title" class="slider-title">min. IVs</span>
+                    <div class="slider-container">
+                      <div id="highlight-iv-slider"></div>
+                    </div>
+                  </div>
+                  <div class="form-control">
+                    <span id="highlight-level-slider-title" class="slider-title">min. Level</span>
```

Comment:
Could you please use i18n on those labels too?

Reply:
Sure...